packages/synapse-react-client/src/synapse-queries/entity/useExportToTerra.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  DownloadPFBRequest,
+  QueryBundleRequest,
+} from '@sage-bionetworks/synapse-types'
+import { displayToast } from '../../components/ToastMessage/ToastMessage'
+import { parseEntityIdFromSqlStatement } from '../../utils/functions/SqlFunctions'
+import { useDownloadTableQueryResultAsPFB } from '../table/useDownloadTable'
+
+export function useExportToTerra(queryBundleRequest: QueryBundleRequest) {
```

Comment:
No idea if this works, may need some additional work (to be tracked in a new ticket)